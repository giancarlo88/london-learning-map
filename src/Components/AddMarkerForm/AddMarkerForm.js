import React from 'react'

const AddMarkerForm = () => <div>Add marker form</div>

export default AddMarkerForm
