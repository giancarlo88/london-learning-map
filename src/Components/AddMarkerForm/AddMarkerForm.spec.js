import { shallow } from 'enzyme'
import React from 'react'
import AddMarkerForm from './AddMarkerForm'

describe(AddMarkerForm.name, () => {
  describe('@renders', () => {
    it('should render', () => {
      const wrapper = shallow(<AddMarkerForm />)
      expect(wrapper).toMatchSnapshot()
    })
  })
})
