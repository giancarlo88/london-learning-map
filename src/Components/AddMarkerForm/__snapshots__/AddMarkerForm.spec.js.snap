// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddMarkerForm @renders should render 1`] = `
<div>
  Add marker form
</div>
`;
